consumption-interface/Release_Notes_1_0_0.md

```diff
@@ -0,0 +1,15 @@
+Known issues and limitations of the Consumption Interface Supervisor Service
+
+LCI : Local Consumption Interface
+
+- LCI is not supported on Stretched Supervisors.
```

Comment:
I don't think this is the official terminology. Maybe `Multi-Zone Supervisors` based on our [vSphere Zones](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-installation-configuration/GUID-7A0AA982-7F0C-40E7-8635-FB45DF0EE218.html) documentation.